config.py

```diff
@@ -67,10 +67,7 @@ class LabelEncodingEnum(Enum):
 # Training variables
 LEARNING_RATE = 0.001
 BATCH_SIZE = 8
-if MODEL == ModelEnum.BASIC_CNN:
-    NB_EPOCHS = 20
-elif MODEL == ModelEnum.PIANOCKTAIL_GRU:
-    NB_EPOCHS = 300
+NB_EPOCHS = 300
```

Comment:
Since we are now passing the model through CLI, this is ignored. I don't think we need to split epochs number in the config anymore. If we want to do it, let's do it in CLI